crates/solver/src/driver.rs

```diff
@@ -204,7 +206,6 @@ impl Driver {
 
     async fn submit_settlement(
         &self,
-        auction_id: u64,
```

Comment:
This was only used for logging. Since we have informational spans that include this, it was redundant and I removed it.